device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ModuleClient.java

```diff
@@ -193,7 +197,41 @@ public static ModuleClient createFromEnvironment(IotHubClientProtocol protocol,
         {
             log.debug("Creating module client with the provided connection string");
 
-            //Codes_SRS_MODULECLIENT_34_020: [If an edgehub or iothub connection string is present, this function shall create a module client instance using that connection string and the provided protocol.]
+            //Check for a different default cert to be used
```

Comment:
During the moduleClient's createFromEnvironment flow, there may be a PEM formatted public certificate that the SDK is expected to trust in order to connect to the Edgehub